docs/04-api-reference/front-commerce-core/react.mdx

```diff
@@ -200,3 +200,40 @@ function AcmePage() {
   );
 }
 ```
+
+## `usePublicConfig`
+
+Returns the public config of the Front-Commerce instance.
+
+```tsx
+import { usePublicConfig } from "@front-commerce/core/react";
+
+function PasswordHint() {
+  const publicConfig = usePublicConfig();
+
+  if (publicConfig?.publicConfig?.password?.disablePasswordHint) {
+    return null;
+  }
+
+  return <button>👁️</button>;
+}
+```
+
+## `getPublicConfig`
+
+The same as `usePublicConfig` it returns the public config of the Front-Commerce
+instance, this can be used outside of a react scope.
```

Comment:
**issue:** when reading the example it's not clear of why using the hook is preferable.

**question:** should we add a note asking to use it as a last resort?

**suggestion:** use another example not related to a React component